BedrockServer.cpp

```diff
@@ -846,6 +847,15 @@ void BedrockServer::runCommand(unique_ptr<BedrockCommand>&& _command, bool isBlo
                 core.prePeekCommand(command);
             }
 
+            uint64_t conflictLockStartTime = 0;
+            if (lastConflictPage) {
+                conflictLockStartTime = STimeNow();
+            }
+            PageLockGuard pageLock(lastConflictPage);
+            if (lastConflictPage) {
+                SINFO("Waited " << (STimeNow() - conflictLockStartTime) << "us for lock on db page " << lastConflictPage << ".");
```

Comment:
Can we make this `ms` like (mostly) everywhere in our logs?

Reply:
Can we track this time separately in our timing logs?

Reply:
I'd rather not. I care more about the usefulness of the number than the consistency of the logs, and these are numbers that may very often be sub 1ms times, and even if they're in the 0-5ms range, we lose almost all our precision by truncating here.

Reply:
I assume that answer was for Flo, can we include this timing as a separate entry in [our timing info log](https://github.com/Expensify/Bedrock/blob/b65e119af182504d3a8871e2a9dd8e85196f22bd/BedrockCommand.cpp#L230)?

Reply:
We could. I'm not sure how useful it will be. Let's try it out and see if the metric seems to add much. I don't really know what we'll do wit that number.